frontend/src/components/SeedlotRegistrationSteps/OrchardStep/index.tsx

```diff
@@ -0,0 +1,570 @@
+              Have biotechnological processes been used to produce this seedlot?
+            </label>
+            <Checkbox
+              id="bio-processes"
+              name="biotechProcess"
+              labelText="No, biotechnological processes have not been used to produce this seedlot"
+              defaultChecked
+              onChange={(event: React.ChangeEvent<HTMLInputElement>) => checkboxesHandler(event)}
+            />
+          </Column>
+        </Row>
+        <Row className="seedlot-orchard-title-row">
+          <Column lg={8}>
+            <h2>Pollen information</h2>
+            <Subtitle text="Enter the pollen contaminant information" />
+          </Column>
+        </Row>
+        <Row className="seedlot-orchard-field">
+          <Column sm={4} md={8} lg={16}>
+            <label htmlFor="pollen-contamination" className="bcgov--label">
+              Was pollen contamination present?
+            </label>
+            <Checkbox
+              id="pollen-contamination"
+              name="noPollenContamination"
+              labelText="No, there was no pollen contamination present in the seed orchard"
+              defaultChecked
+              onChange={(event: React.ChangeEvent<HTMLInputElement>) => checkboxesHandler(event)}
+            />
+          </Column>
+        </Row>
+        <div className={!responseBody.noPollenContamination ? '' : 'seedlot-orchard-hidden'}>
+          <Row className="seedlot-orchard-field">
+            <Column sm={4} md={4} lg={6}>
+              <NumberInput
+                id="pollen-percentage-number-input"
+                name="breedingPercentage"
+                ref={(el: HTMLInputElement) => addRefs(el, 'breedingPercentage')}
+                min={0}
+                max={100}
+                value={0}
+                step={10}
+                disableWheel
+                type="number"
+                label="Contaminant pollen breeding percentage (optional) (%)"
+                helperText="If contaminant pollen was present and the contaminant pollen has a breeding value"
+                invalid={invalidBreeding}
+                invalidText="Please enter a valid value between 0 and 100"
+                onChange={(e: React.ChangeEvent<HTMLInputElement>) => percentageHandler(e)}
+                onBlur={() => validateBreedingPercentage()}
+              />
+            </Column>
+          </Row>
+          <Row className="pollen-methodology-checkbox">
+            <Column sm={4} md={8} lg={16}>
+              <label htmlFor="pollen-methodology" className="bcgov--label">
+                Contaminant pollen methodology
+              </label>
+              <Checkbox
+                id="pollen-methodology"
+                name="pollenMethodology"
+                labelText="Regional pollen monitoring"
+                defaultChecked
+                onChange={(event: React.ChangeEvent<HTMLInputElement>) => checkboxesHandler(event)}
```

Comment:
Current:
![image](https://user-images.githubusercontent.com/25162561/228053099-62e6e8bb-cc65-4677-bcec-a1d2b2969ae6.png)
Design:
![image](https://user-images.githubusercontent.com/25162561/228053293-7aaf8d43-ef3d-40ac-a296-3895ec5b85e2.png)

Not sure why this single checkbox is designed differently than others, probably an oversight from the design team

Reply:
The UX team clarified that for us, this is a read-only field, so the design is supposed to be a bit different.

Reply:
👀 the design team does not make mistakes